plugins/doc_fragments/foreman.py

```diff
@@ -273,6 +273,8 @@ class ModuleDocFragment(object):
       - Only available for Katello installations.
     required: false
     type: str
+extends_documentation_fragment:
```

Comment:
I wonder if that even works.

Reply:
Well, no, of course not. At least not with ansible-doc

Reply:
:disappointed: